### PR TITLE
Enable HSTS by default

### PR DESCRIPTION
**Pull request by roborourke**
Set HSTS to be on by default.

Fixes https://github.com/humanmade/product-dev/issues/986

### CODE REVIEW

**Review by robindevitt — APPROVED** [review]

File: docs/browser.md [hunk 1]
```diff
@@ -181,18 +181,28 @@ In some cases, you may want to adjust or disable these headers depending on the
 
 The [`Strict-Transport-Security` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) (sometimes called HSTS) is used to enforce HTTPS (TLS/SSL) connections when loading a site and can be used to enhance the site's security.
 
-By default, Altis does not enable HSTS. You can set the value of this header manually by defining the `ABS_HSTS` constant:
+By default, Altis enables HSTS by default with the value `max-age=31536000; includeSubDomains`. You can configure the header using the `strict-transport-policy` setting:
```
Comment (mikelittle) [inline docs/browser.md:184]: Lose the second "by default".

File: docs/browser.md [hunk 2]
```diff
@@ -181,18 +181,28 @@ In some cases, you may want to adjust or disable these headers depending on the
...
+```json
+{
+	"browser": {
+		"strict-transport-policy": "max-age=3600"
+	}
+}
 ```
 
-To disable the automatic behaviour entirely, set the constant to `false`:
+You can also switch the header off entirely by setting this to false:
```
Comment (mikelittle) [inline docs/browser.md:194]: "completely" not "entirely" as you have switched verb and from passive to active.

File: docs/browser.md [hunk 3]
```diff
@@ -181,18 +181,28 @@ In some cases, you may want to adjust or disable these headers depending on the
...
-define( 'ABS_HSTS', false );
+```json
+{
+	"browser": {
+		"strict-transport-policy": false
+	}
+}
 ```
 
+Finally, if you set the value to `null` then Altis will send the header if the current request is using SSL.
```
Comment (mikelittle) [inline docs/browser.md:204]: "will send" or "will not send" when it is null?

**Review by mikelittle — CHANGES_REQUESTED** [review]

File: docs/browser.md (hunk 3 above)
Comment (roborourke) [reply docs/browser.md:204]: Will send, the browser security plugin reverts to a conditional check. Thinking I'll update the wording a bit though:
```suggestion
Finally, if you set the value to `null` then Altis _will_ send the header but only if the current request is already using HTTPS.
```

File: docs/browser.md (hunk 2 above)
Comment (roborourke) [reply docs/browser.md:194]:
```suggestion
You can also switch the header off completely by setting this to false:
```

File: docs/browser.md (hunk 1 above)
Comment (roborourke) [reply docs/browser.md:184]:
```suggestion
By default, Altis enables HSTS with the value `max-age=31536000; includeSubDomains`. You can configure the header using the `strict-transport-policy` setting:
```

**Review by mikelittle — APPROVED** [review]
Looks good to me

**Pull request merged**